src/framework/ui/input/input.component.tsx

```diff
@@ -280,6 +290,16 @@ export class InputComponent extends React.Component<InputProps> {
     };
   };
 
+  private renderIconTouchableElement = (style: StyleType): React.ReactElement<TouchableWithoutFeedbackProps> => {
+    const iconElement: IconElement = this.renderIconElement(style);
+
+    return (
+      <TouchableWithoutFeedback onPress={this.onIconPress}>
```

Comment:
Why not the TouchableOpacity with "activeOpacity={1.0}"?

Reply:
This is the same but without a need to apply `style` property